src/main/java/by/exadel/internship/dto/internshipDTO/GuestInternshipDTO.java

```diff
@@ -62,4 +62,7 @@
 
     @NotNull
     private LocalDate registrationEndDate;
+
+    @NotNull
+    private Boolean deleted;
```

Comment:
UI need this field?

Reply:
I hope not. In this case remove this field in other DTO as well.

Reply:
Fixed